linux-bootstrap-1.md

```diff
@@ -54,24 +53,24 @@ But if we take the biggest segment part and offset: `0xffff:0xffff`, it will be:
 '0x10ffef'
 ```
 
-which is 65519 bytes over first megabyte. Since only one megabyte is accessible in real mode, `0x10ffef` becomes `0x00ffef` with disabled [A20](http://en.wikipedia.org/wiki/A20_line).
+which is 65519 bytes over the first megabyte. Since only one megabyte accessible in real mode, `0x10ffef` becomes `0x00ffef` with disabled [A20](http://en.wikipedia.org/wiki/A20_line).
 
-Ok, now we know about real mode and memory addressing, let's get back to register values after reset.
+Ok, now we know about real mode and memory addressing, let's back to registers values after reset.
```

Comment:
the same https://github.com/0xAX/linux-insides/pull/7/files#diff-054c5daca2ffd0c4037409522c210d3cR59